diff --git a/node_modules/lemmy-bot/dist/bot.js b/node_modules/lemmy-bot/dist/bot.js
index 4b68345..42a3a25 100644
--- a/node_modules/lemmy-bot/dist/bot.js
+++ b/node_modules/lemmy-bot/dist/bot.js
@@ -380,10 +380,7 @@ _LemmyBot_isRunning = new WeakMap(), _LemmyBot_instance = new WeakMap(), _LemmyB
                 const timeout = setTimeout(() => {
                     __classPrivateFieldGet(this, _LemmyBot_instances, "m", _LemmyBot_runChecker).call(this, checker, secondsBetweenPolls);
                     __classPrivateFieldSet(this, _LemmyBot_timeouts, __classPrivateFieldGet(this, _LemmyBot_timeouts, "f").filter((t) => t !== timeout), "f");
-                }, 1000 *
-                    (secondsBetweenPolls < DEFAULT_SECONDS_BETWEEN_POLLS
-                        ? DEFAULT_SECONDS_BETWEEN_POLLS
-                        : secondsBetweenPolls));
+                }, 1000 * secondsBetweenPolls);
                 __classPrivateFieldGet(this, _LemmyBot_timeouts, "f").push(timeout);
             }
             else if (__classPrivateFieldGet(this, _LemmyBot_credentials, "f") && !__classPrivateFieldGet(this, _LemmyBot_auth, "f")) {
